ember-core/shared/src/main/scala/org/http4s/ember/core/h2/H2Keys.scala

```diff
@@ -32,7 +32,7 @@ object H2Keys {
   // which means immediately using http2 without any upgrade mechanism
   // but is invalid if the receiving server does not support the
   // mechanism.
-  @deprecated(message = "Use org.http4s.h2.H2Keys.Http2PriorKnowledge instead", since = "0.23.27")
+  @deprecated(message = "Use org.http4s.h2.H2Keys.Http2PriorKnowledge instead", since = "0.23.28")
```

Comment:
Wow, I'm both impressed and confused to see that PR.
And it's a good candidate for moving into scala-steward config.

Reply:
I just found similar in #7523.